unified-runtime/source/adapters/level_zero/v2/command_buffer.cpp

```diff
@@ -29,6 +31,167 @@ void checkImmediateAppendSupport(ur_context_handle_t context) {
 
 } // namespace
 
+ur_result_t ur_exp_command_buffer_handle_t_::updateKernelSizes(
```

Comment:
Can this code be shared with v1 adapter? It's copying large sections of `updateCommandBuffer` in the v1 adapter